tests/ragger/test_domain_name.py

```diff
@@ -30,21 +30,41 @@ def verbose_fixture(request) -> bool:
     return request.param
 
 
-def common(firmware: Firmware, app_client: EthAppClient) -> int:
+def common_challenge(firmware: Firmware, app_client: EthAppClient) -> int:
 
     if firmware.device == "nanos":
         pytest.skip("Not supported on LNS")
     challenge = app_client.get_challenge()
     return ResponseParser.challenge(challenge.data)
 
 
+def common_nav(firmware: Firmware,
```

Comment:
If you want you can use the simple navigation helper from ragger. It does just that

Reply:
https://github.com/LedgerHQ/app-boilerplate/pull/119/files#diff-833eec1ba57c6e4d41959eb8ae23f6b8c0709eef144acf1445937b0342753dd6

Reply:
Yes, it is already the case with the up-to-date `develop` branch.
This commit has been skipped now, after the rebase.